handbook/company.md

```diff
@@ -262,6 +262,131 @@ Groups are organized around goals. Connecting people with the same goals helps t
+- Platform PM: Zach Wasserman
+- Agent PM: TBD (Mo Zhu?)
+
+Each group is associated with an engineering manager (EM), who, with their group of engineers, form the engineering members of the group.
+
+Each group's PM works very closely with engineers within their group:
+- The PM **prioritizes** work and defines **what** to iteratively build and release next, within their group's domain, to best serve the group's goals and the goals of the company as a whole. The PM communicates **why** this work is prioritized and works with engineering to come up with the best possible **how**.
+- The EM (along with engineers) defines **how** to implement that definition within the surface area of code, processes, and rituals owned by their group, while serving their group’s goals and the goals of the company as a whole.
+
+### Interface group
+#### Responsibilities
+- Everything related to Fleet's graphical user interface (besides for the desktop application portion of Fleet Desktop)
+- `fleetctl` (the Fleet command-line interface) and the associated yaml documents (almost everything in fleetctl besides the `fleetctl debug` subcommands)
+- The REST API that serves these
+- The UX/developer experience, flow, steps, and associated UI and API interfaces for how integrations that require any user interaction or configuration (e.g. GeoIP, Zendesk, Jira), including which 3rd party integrations are supported and which API styles and versions are chosen
+- End to end testing of the application (eg. Cypress)
+- The REST API documentation
+- Future officially-supported wrapper SDKs, such as the Postman collection, or e.g. a Python SDK
+- Fleet's configuration surface, including
+  - The config settings that exist for Fleet deployments, and how they're configured
+  - How feature flags are used
+  - Their default values, supported data types, and error messages
+  - Associated documentation on fleetdm.com
+  - The UX of upgrading and downgrading and sidegrading Fleet tiers, and managing license keys
+
+#### Consumers
+- A human using Fleet's graphical user interface
+- A human who is writing code that integrates Fleet's REST API 
+- A human reading Fleet's REST API docs
+- A human using fleetctl, Fleet's Postman collection, or Fleet's other future SDK wrappers
+
+These humans might be working within the "Interface" group itself, insofar as they consume the Fleet REST API.  Or they might be a contributor in the Fleet community.  Or one of Fleet's core users or customers, usually in an SRE, IT, or security role in an organization.
+
+#### Goals
+- Bring value to Fleet users through delivery of new features and iteration on existing features.
+- Increase adoption and stickiness of features.
+- Keep the graphical user interface, REST API, fleetctl, and SDKs like Postman reliable, minimal, consistent, and easy to use.
+- Ensure observance of semantic versioning for the Fleet API and config between releases, so that only major versions include breaking changes.
```

Comment:
>semantic versioning

We [won't be using](https://fleetdm.com/docs/contributing/api-versioning#why-not-semantic-versioning) semantic versioning.

We're taking [inspiration from Stripe](https://stripe.com/docs/api/versioning) and using date versioning.

Reply:
Here's a great [Stripe blog post](https://stripe.com/blog/api-versioning) discussing why they chose date versioning.

Reply:
Good point. Fixing this to indicate that it's about not breaking the API except as communicated.